src/Hazelcast.Net/DistributedObjects/IHMap.Setting.cs

```diff
@@ -26,19 +27,25 @@ public partial interface IHMap<TKey, TValue> // Setting
         /// <param name="key">A key.</param>
         /// <param name="value">A value.</param>
         /// <returns>A task that will complete when the entry has been added or updated.</returns>
+        /// <remarks>
+        /// <para>The value has an infinite time-to-live, and never becomes idle.</para>
+        /// </remarks>
         Task SetAsync(TKey key, TValue value);
 
         /// <summary>
         /// Sets (adds or updates) an entry.
         /// </summary>
         /// <param name="key">A key.</param>
         /// <param name="value">A value.</param>
-        /// <param name="timeToLive">A time to live.</param>
+        /// <param name="timeToLive">A time to live (<see cref="TimeToLive.Infinite"/> i.e. 0ms to live
+        /// forever; <see cref="TimeToLive.Default"/> i.e. -1ms to use the server-configured value).</param>
         /// <returns>A task that will complete when the entry has been added or updated.</returns>
         /// <remarks>
         /// <para>The value is automatically expired, evicted and removed after the
-        /// <paramref name="timeToLive"/> has elapsed.</para>
-        /// TODO: document zero & infinite
+        /// <paramref name="timeToLive"/> has elapsed. If <paramref name="timeToLive"/> is <see cref="TimeToLive.Infinite"/>
+        /// i.e. -0ms, the value is retained indefinitely. If it is <see cref="TimeToLive.Default"/> i.e. -1ms,
```

Comment:
Minor: `-0ms` -> `0ms`. There are also a couple of such places below

Reply:
fixing